File: EventCheckin.ClientApp/src/app/api/models/ticket-pass-model.ts
/* tslint:disable */
/* eslint-disable */
import { EventModel } from '../models/event-model';
import { UserModel } from '../models/user-model';
export interface TicketPassModel {
  allowedGuestCount?: number;
  allowedParkingCount?: number;
  assignedBy?: number;
  assignedDateUtc?: string | null;
  enteredOTP?: string | null;
  enteredParkingOTP?: string | null;
  entryOTP?: string | null;
  entryOTPTime?: string | null;
  entrySecurity?: string | null;
  entryStatus?: number;
  entryStatusName?: string | null;
  eventId?: number;
  eventModel?: EventModel;
  id?: number;
  isActive?: boolean;
  isInviteSelected?: boolean;
  isParkingAllowed?: boolean;
  isParkingSelected?: boolean;
  name?: string | null;
  parkSecurity?: string | null;
  parkStatus?: number;
  parkStatusName?: string | null;
  parkingOTP?: string | null;
  parkingOTPTime?: string | null;
  passDay?: string | null;
  passFromTime?: string | null;
  passToTime?: string | null;
  phoneNumber?: string | null;
  ticketNo?: string | null;
  userId?: number;
  userModel?: UserModel;
}

File: EventCheckin.Api/Models/Events/TicketPassModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace EventCheckin.Api.Models.Events
{
    public class TicketPassModel
    {
        public long Id { get; set; }
        
        public long EventId { get; set; }

        public string TicketNo { get; set; }

        public bool IsActive { get; set; }

        public int AllowedGuestCount { get; set; }

        public long UserId { get; set; }

        public string EntryOTP { get; set; }

        public DateTime? EntryOTPTime { get; set; }

        public string EntrySecurity { get; set; }

        public int EntryStatus { get; set; }

        public string EntryStatusName { get; set; }

        public bool IsParkingAllowed { get; set; }

        public int AllowedParkingCount { get; set; }

        public string ParkingOTP { get; set; }

        public DateTime? ParkingOTPTime { get; set; }

        public string ParkSecurity { get; set; }

        public int ParkStatus { get; set; }

        public string ParkStatusName { get; set; }

        public long AssignedBy { get; set; }

        public DateTime? AssignedDateUtc { get; set; }

        public string PassDay { get; set; }

        public string PassFromTime { get; set; }

        public string PassToTime { get; set; }

        public string PhoneNumber { get; set; }

        public string Name { get; set; }

        public UserModel UserModel { get; set; }

        public EventModel EventModel { get; set; }

        public bool IsInviteSelected { get; set; }

        public bool IsParkingSelected { get; set; }
    }
}
